### PR TITLE
Allow the blobstore to integrate with a GCS server

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2162/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2164/

File: lib/constants.py [hunk 1]
```diff
@@ -105,3 +105,9 @@ class HTTPCodes(object):
 
 # The default log directory for AppScale services.
 LOG_DIR = os.path.join('/var', 'log', 'appscale')
+
+# The number of seconds to wait before retrying some operations.
+SMALL_WAIT = 5
+
+# The number of seconds to wait before retrying some operations.
+TINY_WAIT = .1
```
Comment (menivaitsi) [inline lib/constants.py]: 😆

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Blobstore/1378/

**Pull request merged**